### ISSUE TITLE
MO2 Alpha3 not work Fallout4 1.3.45beta

### ISSUE THREAD

**Issue by amigajboy** [TanninOne/modorganizer#502]
I cannot attach ModOrganizer.exe.dmp.zip for some reason.
So I ride only log.

mo_interface.log

17:44:07 [D] start main application
17:44:07 [D] data path: C:/ModOrganizer2
17:44:07 [D] Working directory: C:\ModOrganizer2
17:44:07 [D] initializing core
17:44:07 [D] usvfs log messages are written to C:/ModOrganizer2/logs/usvfs-2016-01-16_08-44-07.log
17:44:07 [D] initialize plugins
17:44:07 [D] looking for plugins in C:\ModOrganizer2\plugins
17:44:07 [D] loaded plugin "check_fnis.dll"
17:44:07 [D] loaded plugin "diagnose_basic.dll"
17:44:07 [D] loaded plugin "gameFallout3.dll"
17:44:07 [D] loaded plugin "gameFallout4.dll"
17:44:07 [D] loaded plugin "gameFalloutNV.dll"
17:44:07 [D] loaded plugin "gameOblivion.dll"
17:44:07 [D] loaded plugin "gameSkyrim.dll"
17:44:07 [D] loaded plugin "inibakery.dll"
17:44:07 [D] loaded plugin "inieditor.dll"
17:44:07 [D] loaded plugin "installer_bain.dll"
17:44:07 [D] loaded plugin "installer_bundle.dll"
17:44:07 [D] loaded plugin "installer_fomod.dll"
17:44:07 [D] loaded plugin "installer_manual.dll"
17:44:07 [D] loaded plugin "installer_ncc.dll"
17:44:07 [D] loaded plugin "installer_quick.dll"
17:44:07 [C] Failed to load python runner ():  (errorcode 193)

**Comment by Percephere** [comment]
I'll test this with 1.3.47 (current beta) and report back

**Comment by Percephere** [comment]
I've got  1.3.47 working alright for me. I did however notice that Steam sometimes tries to check the version (since it is a beta release) before launching, which either causes FO4 to hang up indefinitely or crash MO. I managed to get by that by launching FO4 normally via Steam without any mods. The only issue I did notice was that the mouse cursor remained over the game, but that may be an issue with the game itself.

Make sure F4SE is updated to v0.1.5 if you're playing with the beta patch.
